docs/source/cookbook/dataframes.rst

```diff
@@ -0,0 +1,166 @@
+Using external data
+-------------------
+
+It's not always practical or desirable to store all the data required by a model in the JSON document. This is particularly true when working with large amounts of timeseries data, but is also applicable to data better stored in a tabular format.
+
+Many parameter types support loading data from an external file using the ``"url"`` keyword. The data is loaded during the setup/reset process, before the model is run.
+
+Supported formats
+=================
+
+Support for the following data formats is available via the `Pandas <http://pandas.pydata.org/pandas-docs/stable/io.html>`_ module.
```

Comment:
We should make a note here that other non-documented keys in the JSON get passed to the corresponding `pandas.read_xxx` function.